examples/text_summarization/pointer_summarizer/utils/data.py

```diff
@@ -0,0 +1,183 @@
+# Most of this file is copied form https://github.com/abisee/pointer-generator/blob/master/data.py
```

Comment:
Please unify the coding style here, and make sure the License and author is clear.